authors/apps/profiles/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from authors.apps.authentication.models import User
+from authors.apps.core.models import TimeStamp
+
+
+class Profile(TimeStamp):
```

Comment:
I don't think this is the correct way of using inheritance. The parent class should be a class that represents a general model of the child that inherits it. This line implies that a Profile is a type of TimeStamp. It's better to add in the `created_at` and `modified_at` directly to the model, or if you really want the automation, you can look into decorators

Reply:
Thank you